src/interface/web/app/components/navMenu/navMenu.tsx

```diff
@@ -89,31 +95,34 @@ export default function NavMenu() {
     }
 
     return (
-        <div className={styles.titleBar}>
-            {showLoginPrompt && (
-                <LoginPrompt
-                    onOpenChange={setShowLoginPrompt}
-                    isMobileWidth={isMobileWidth}
-                    loginRedirectMessage={"Login to your second brain"}
-                />
-            )}
-            {isMobileWidth ? (
+        <SidebarMenu className="border-none p-0 m-0">
+            <SidebarMenuItem className="p-0 m-0">
                 <DropdownMenu>
-                    <DropdownMenuTrigger>
-                        {userData ? (
-                            <Avatar
-                                className={`h-10 w-10 border-2 ${userData.is_active ? "border-yellow-500" : "border-stone-700 dark:border-stone-300"}`}
-                            >
-                                <AvatarImage src={userData?.photo} alt="user profile" />
-                                <AvatarFallback className="bg-transparent hover:bg-muted">
-                                    {userData?.username[0].toUpperCase()}
-                                </AvatarFallback>
-                            </Avatar>
-                        ) : (
-                            <UserCircle className="h-10 w-10" />
-                        )}
+                    <DropdownMenuTrigger asChild>
+                        <SidebarMenuButton className="p-0 m-0 rounded-lg" asChild>
+                            {userData ? (
+                                <span className="flex items-center gap-2">
+                                    <Avatar
+                                        className={`${sideBarIsOpen ? "h-8 w-8" : "h-6 w-6"} border-2 ${userData.is_active ? "border-yellow-500" : "border-stone-700 dark:border-stone-300"}`}
+                                    >
+                                        <AvatarImage src={userData?.photo} alt="user profile" />
+                                        <AvatarFallback className="bg-transparent hover:bg-muted">
+                                            {userData?.username[0].toUpperCase()}
+                                        </AvatarFallback>
+                                    </Avatar>
+                                    {sideBarIsOpen && (
+                                        <>
+                                            <p>{userData?.username}</p>
+                                            <ChevronUp className="w-6 h-6" />
```

Comment:
Ideally the Chevron up should always be left aligned in the sidepane. Otherwise it aligns with the length of the username
```suggestion
                                            <ChevronUp className="w-6 h-6 ml-auto" />
```

Reply:
Fixed in #1015